### DISCUSSION TITLE
Port Connectors

### DISCUSSION THREAD

**Discussion by dwfunk4475** [gizmo-platform/gizmo#42] [Ideas]
Please! Please! Please! Come up with a better connctor scheme.  The slightest bump, look at them wrong and the connectors come out. The kids lost the motor arm because the connector came loose. We tried to hold them down with electrical tape, but that failed.

**Comment by Vaelatern** [comment]
Interesting point! I wonder: if you put the electrical tape over the Gizmo ports, and plugged in through the tape (forcing the pins through) if the tape would help hold the pins in "from below"

**Comment by the-maldridge** [comment]
After reviewing the many, many ways people secured connectors to their Gizmos last year, we've found that [this design](https://www.printables.com/model/1219512-gizmo-verson-1011-case-2025) from printables is the most reliable, and is our recommendation going forward.  The inner ledge constrains the connectors, while the outer component provides a place to take a strip of blue-tape (permitted by the rules for RK components) to retain the connectors.  All hubs have been advised to implement this or similar as a feature of their cases.
